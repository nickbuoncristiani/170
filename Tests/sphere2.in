66
25
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66
46 66 60 54 16 9 49 30 27 65 57 10 11 1 34 26 37 47 39 51 3 33 64 53 61
15
x 1.16019 1.60709 1.36 x 1.2076 1.0644 1.76639 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.16019 x 1.60993 1.94484 1.51184 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.60709 1.60993 x x 1.1464 1.09053 x x 1.02549 1.99201 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.36 1.94484 x x 1.6043 x 1.91885 x x x 1.41636 1.94406 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x 1.51184 1.1464 1.6043 x x x x 1.22063 x 1.34891 x 1.15227 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.2076 x 1.09053 x x x x 1.33281 x 1.80929 x x x 1.37972 1.94962 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.0644 x x 1.91885 x x x 1.97984 x x x 1.31815 x x x 1.29002 1.57786 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.76639 x x x x 1.33281 1.97984 x x x x x x 1.05658 x 1.42747 x x x x x 1.64292 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x 1.02549 x 1.22063 x x x x 1.1357 x x 1.71998 x x x x 1.18783 1.16021 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x 1.99201 x x 1.80929 x x 1.1357 x x x x x 1.84468 x x 1.2135 x x x x 1.14141 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x 1.41636 1.34891 x x x x x x 1.78066 1.40509 x x x x x x 1.53941 1.52697 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x 1.94406 x x 1.31815 x x x 1.78066 x x x x x 1.40904 x x 1.75178 x x x 1.36894 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x 1.15227 x x x 1.71998 x 1.40509 x x x x x x x 1.18428 x 1.49301 x x x 1.36063 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x 1.37972 x 1.05658 x x x x x x 1.75139 x x x x x x 1.55698 x x x 1.2391 1.03662 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x 1.94962 x x x 1.84468 x x x 1.75139 x x x x x x x x 1.18086 x x x 1.67768 x x 1.10056 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.29002 1.42747 x x x x x x x x 1.59574 x x x x 1.23461 x x x x x 1.3374 1.30948 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.57786 x x x x 1.40904 x x x 1.59574 x x x x x x x 1.10594 x x x x 1.78789 x 1.77244 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x 1.18783 1.2135 x x x x x x x x 1.57541 x x x 1.42312 x x x x x x x x 1.59414 1.94317 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x 1.16021 x x x 1.18428 x x x x 1.57541 x x x x x x 1.74525 x x x x x x x 1.01654 x x 1.75243 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.53941 1.75178 x x x x x x x x 1.17185 x x 1.57021 x x x x x x x x x 1.89439 1.83024 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.52697 x 1.49301 x x x x x x 1.17185 x x x x 1.08715 x x x x x x x x x 1.02851 x 1.05681 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x 1.64292 x x x x x 1.55698 x 1.23461 x x x x x x x x x 1.38982 x 1.21427 x x x x x x x x x 1.18109 x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x 1.14141 x x x x 1.18086 x x 1.42312 x x x x x x x x x x x 1.81667 x 1.67517 x x x x x x 1.65889 x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x 1.36894 x x x x 1.10594 x x 1.57021 x x x x x x x x x x 1.97618 x x 1.28358 x x x x x 1.31579 x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x 1.36063 x x x x x 1.74525 x 1.08715 x x x x x x x x x x x x x x 1.81707 1.52595 x x x 1.17642 x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x 1.2391 x x x x x x x 1.38982 x x x x 1.9966 x x x x x x x x x x 1.40484 x x x 1.63502 1.15797 x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x 1.03662 1.67768 x x x x x x x x x x 1.9966 x x x 1.6549 x x x x x x x x x x x x 1.73984 x 1.17 x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x 1.3374 x x x x x 1.21427 x x x x x x 1.19258 x x x x x x x x 1.45739 x x x 1.63001 x 1.56561 x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x 1.30948 1.78789 x x x x x x x x x x 1.19258 x x 1.64568 x x x x x x x x x x x x 1.05316 x 1.11408 x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.10056 x x x x x x x 1.81667 x x x 1.6549 x x x x x x x x x x x 1.02319 x x x x x 1.47819 x 1.66241 x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x 1.77244 x x x x x x 1.97618 x x x x 1.64568 x x x x x x x x x x 1.1004 x x x x x 1.58165 x 1.04462 x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x 1.59414 x x x x 1.67517 x x x x x x x x x 1.80559 x x x x x 1.10082 x x x x x x x 1.63897 x 1.63325 x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x 1.94317 1.01654 x x x x x x x x x x x x 1.80559 x x x 1.44354 x x x x x x x x x x x x 1.53305 x 1.6157 x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x 1.89439 x x x 1.28358 x x x x x x x x x x 1.27349 x x x x 1.38245 x x x x x x x 1.6348 x 1.57479 x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x 1.83024 1.02851 x x x x x x x x x x x x 1.27349 x x 1.12244 x x x x x x x x x x x x 1.45818 x 1.4523 x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x 1.75243 x x x x x 1.81707 x x x x x x x 1.44354 x x x x x x x 1.00509 x x x x x x x x x 1.33956 x 1.4315 x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x 1.05681 x x x 1.52595 x x x x x x x x x 1.12244 x x x x x 1.69753 x x x x x x x x x x 1.78322 1.05688 x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x 1.18109 x x x 1.40484 x 1.45739 x x x x x x x x x x x x x 1.17061 x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x 1.65889 x x x x x x 1.02319 x 1.10082 x x x x x x x x x x x x x x 1.50039 x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x 1.31579 x x x x x x 1.1004 x x 1.38245 x x x x x x x x x x x x x 1.36382 x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x 1.17642 x x x x x x x x x x 1.00509 1.69753 x x x x x x x x x x x x x x x 1.90311 x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x 1.63502 x 1.63001 x x x x x x x x x 1.17061 x x x x 1.82691 1.09719 x x x x x x x x x 1.89352 x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x 1.15797 1.73984 x x x x x x x x x x x x x x 1.82691 x x 1.0409 x x x x x x x x 1.25352 x x x 1.21348 x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.56561 1.05316 x x x x x x x x x x x x 1.09719 x x x 1.55722 x x x x x x x 1.62613 x x x x 1.01954 x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x 1.17 x x 1.47819 x x x x x x x x x x x x 1.0409 x x x 1.18154 x x x x x x x 1.36541 x x 1.79392 x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.11408 x 1.58165 x x x x x x x x x x x x 1.55722 x x x 1.81898 x x x x x x x 1.7936 x x 1.29746 x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.66241 x 1.63897 x x x x x x 1.50039 x x x x x 1.18154 x x x 1.51892 x x x x x 1.66943 x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.04462 x x 1.6348 x x x x x 1.36382 x x x x x 1.81898 x x x 1.76258 x x x x x 1.08387 x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.63325 1.53305 x x x x x x x x x x x x x 1.51892 x x x 1.40723 x x x 1.86296 x x x x 1.479 x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.57479 1.45818 x x x x x x x x x x x x 1.76258 x x x 1.70747 x x x 1.42782 x x x x 1.13967 x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.6157 x x 1.33956 x x x x x x x x x x x x 1.40723 x x x 1.72657 x x x 1.57447 x x 1.60535 x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.4523 x 1.78322 x x x x x x x x x x x x 1.70747 x x 1.43933 x x x 1.90917 x x x 1.53382 x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.4315 1.05688 x x x 1.90311 x x x x x x x x x 1.72657 1.43933 x x x x 1.89235 x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.89352 1.25352 1.62613 x x x x x x x x x x x x x 1.8689 1.50017 x x 1.75768 x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.36541 x 1.66943 x 1.86296 x x x x x x x x 1.98681 x 1.55077 x x 1.18372 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.7936 x 1.08387 x 1.42782 x x x x x x x x 1.34819 x 1.67495 x x 1.57983 x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.57447 1.90917 1.89235 x x x x x x 1.64319 1.87551 x x x 1.04998 x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.21348 x 1.79392 x x x x x x x x 1.8689 1.98681 x x x x x x 1.87242 1.48513 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.01954 x 1.29746 x x x x x x x 1.50017 x 1.34819 x x x x x 1.19524 x 1.40619 x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.479 x 1.60535 x x x 1.55077 x 1.64319 x x x x x 1.5572 x 1.52992 x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.13967 x 1.53382 x x x 1.67495 1.87551 x x x x x x 1.00336 1.05835 x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.75768 x x x 1.87242 1.19524 x x x 1.5107 1.57383 x 1.57196
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.18372 x x 1.48513 x 1.5572 x 1.5107 x x 1.52923 1.84831
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.57983 x x 1.40619 x 1.00336 1.57383 x x 1.71234 1.1604
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.04998 x x 1.52992 1.05835 x 1.52923 1.71234 x 1.58926
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.57196 1.84831 1.1604 1.58926 x
