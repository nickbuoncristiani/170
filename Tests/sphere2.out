15 10 3 1 3 9 10 15 27 26 42 54 43 27 30 39 47 49 32 23 15
11
10 10
3 3
1 1 46 16
9 9
27 27
26 26
54 54
30 30
39 39
47 47
49 49 60 65
